January 2023 newsletter (#346)
readthedocs/ethicalads.io: content/posts/newsletter-january-2023.md

Title: EthicalAds Newsletter - January 2023
Date: February 8, 2023
description: Updates from January 2023, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community, build-in-public
authors: David Fischer
image: /images/posts/2023-january-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/fr/@inayali?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Ali Inay</a> on <a href="https://unsplash.com/photos/yNaGxHqjOuw?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

These are the major features we released in January:

* We made numerous reporting and performance improvements
  to our advertiser and publisher platform.

> * Our [ML model](https://github.com/readthedocs/ethicalads-model) got its own repository
>   so it's easier to update independently of our platform.

Reviewer: ```suggestion
* Our ML model has been updated to improve performance. It has also been migrated into it's own private repository, so that we can iterate on it without compromising data privacy of our users. 
```

> * Our ML model has been updated to improve performance. It has also been migrated into it's own private repository, so that we can iterate on it without compromising data privacy of our users. 

Author: ```suggestion
* Our ML model has been updated to improve performance. It has also been migrated into its own private repository, so that we can iterate on it without compromising data privacy of our users. 
```

* We had **147 publishers** on our network with at least one paid ad impression.
* We had **12,686,867** paid ad views across the world.

As we expected from the guidance in our [December newsletter]({filename}../posts/newsletter-december-2022.md),
January was a rough month for advertising and losing one of our biggest clients will do that.

However, we expect February to be a significantly better month from an ad sales perspective
although not as good as some of our best months ever at the end of last year.
With the price reduction we put in place for Q1, we have multiple advertisers
increasing the size of their campaigns.
While ad budgets are tight and despite economic gloom in the news,
we haven't seen that much spillover into actual campaigns.

## Upcoming features

The major features in our upcoming roadmap for February:

* We aim to update our advertiser guide with more examples of successful ads
  and concrete ways to track and measure the success of your ad campaigns
  (without compromising privacy).
* We aim to support GitHub Sponsors as a payout option for open source projects.
* We are working on some changes to how we pace ads over the day and over a campaign.

>   We believe there are a number of optimizations to help campaigns perform a little better.

Reviewer: ```suggestion
  We believe there are a number of optimizations to help campaigns perform a little better.
  * We're considering additional product changes that help advertisers optimize their campaigns effectively on our platform over the length of the campaign.
```

Teaser :)
